avm/res/compute/virtual-machine/main.bicep

```diff
@@ -160,9 +160,22 @@ param extensionAadJoinConfig object = {
 }
 
 @description('Optional. The configuration for the [Anti Malware] extension. Must at least contain the ["enabled": true] property to be executed.')
-param extensionAntiMalwareConfig object = {
-  enabled: false
-}
+param extensionAntiMalwareConfig object = osType == 'Windows'
+  ? {
+      enabled: true
+      settings: {
```

Comment:
Question @rahalan, would you think it makes sense to define these settings as a default where they're deployed and just keep the 'enabled'/'disabled' here?

Reply:
@AlexanderSehr , done.